be/src/product/entities/product.entity.ts

```diff
@@ -0,0 +1,32 @@
+import { ObjectType, Field, Int } from '@nestjs/graphql';
+import { Category } from 'src/category/entities/category.entity';
+import { Item } from 'src/item/entities/item.entity';
+import { Column, PrimaryGeneratedColumn, OneToMany, Entity } from 'typeorm';
+
+@ObjectType()
+@Entity('Product')
+export class Product {
+  @PrimaryGeneratedColumn()
+  @Field(() => Int)
+  id: number;
+
+  @Column()
+  @Field(() => String)
+  name: string;
+
+  @Column()
+  @Field(() => String)
+  description: string;
+
+  @Column({type: "json"})
+  @Field(() => String)
+  specs: string;
```

Comment:
I think specs can be nullable - not all products might need to specify specs. (will also have to update inputs)